spring-cloud-gcp-data-spanner/src/main/java/com/google/cloud/spring/data/spanner/core/mapping/SpannerMappingContext.java

```diff
@@ -43,6 +46,19 @@
 
   private ApplicationContext applicationContext;
 
+  private Gson gson;
+
+  public SpannerMappingContext() {
+  }
+
+  public SpannerMappingContext(Gson gson) {
```

Comment:
Could you add a non-null check here? And then `getGson()` can be annotated with `@NonNull`, so the invocations are known to be safe.